dashboard-ui/src/components/SurveyResults/surveyResults.jsx

```diff
@@ -76,7 +76,7 @@ function ScenarioGroup({ scenario, data, version }) {
     }, [data]);
 
     return (<div className='scenario-group'>
-        <h2 className='scenario-header'>Scenario {scenario}</h2>
+        <h2 className='scenario-header'>Scenario {scenarioIndices[scenario]}</h2>
```

Comment:
This duplicates 'scenario' when there is no name since you set the string to "scenario [index]" later on, so it becomes "scenario scenario index".